docs/embedding/tenants.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Tenants
+---
+
+# Tenants
+
+{% include plans-blockquote.html feature="Tenants" is_plural=true %}
+
+Tenants simplify user provisioning and configuration, especially in embedding contexts.
```

Comment:
Think we should say what tenants are first. And maybe emphasize the value proposition. The list already nicely explains why you'd use them.


```suggestion
A Tenant is a named set of attributes you can assign to a user to isolate them from other tenants. If you're building a SaaS app with embedded Metabase dashboards, you can assign the customers of your SaaS to tenants. 

The big advantage of grouping users into tenants is that you can use the same content and group permissions for all tenants, so you don't have to create all new stuff for each tenant, and each tenant only ever sees their own data.
```